apps/prairielearn/src/pages/partials/questionScorePanel.ejs

```diff
@@ -14,8 +14,10 @@
       <% } %>
       <% if (assessment.type == 'Homework') { %>
       <tr>
-        <td>History:</td>
-        <td><%- include('../partials/questionAwardedPoints', {awarded_points: instance_question.variants_points_list, init_points: assessment_question.init_points});  %></td>
+        <td colspan="2" style="white-space: normal;">
```

Comment:
Is the `white-space: normal` necessary here?

Reply:
Yep, without it you end up with this:

<img width="600" alt="image" src="https://github.com/PrairieLearn/PrairieLearn/assets/875833/75737c16-1f87-480d-8391-7ec8202f91d0">

Reply:
What about adding [the text-wrap class](https://getbootstrap.com/docs/4.5/utilities/text/#text-wrapping-and-overflow)?

Reply:
Good call, that does exactly the same thing. Changed to `.text-wrap` in 4009693c923a4ca707d4aff75f3f5a9dedd392ef